projects/use/useop-openh264-config.cmake

```diff
@@ -0,0 +1,28 @@
+# Get the path to the overseer pro install
+get_filename_component(OP_ROOTDIR ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUTE)
+get_filename_component(OP_ROOTDIR ${OP_ROOTDIR} ABSOLUTE)
+
+unset(OPENH264_INCLUDE_DIR CACHE)
+unset(OPENH264_LIBRARY_DIR CACHE)
+
+set(OPENH264_INCLUDE_DIR ${OP_ROOTDIR}/include/wels)
+set(OPENH264_LIB_DIR ${OP_ROOTDIR}/lib)
+
+# determine OP_BUILD_STATIC
+include(${CMAKE_CURRENT_LIST_DIR}/opopts.cmake)
+
+string(TOLOWER ${CMAKE_BUILD_TYPE} build_type)
+
+if(OP_BUILD_STATIC)
+  if(WIN32)
```

Comment:
Yes - this still holds true for 64-bit builds, this flag is true if the build is on Windows (see https://cmake.org/Wiki/CMake_Useful_Variables)